common/src/main/java/com/wynntils/features/redirects/ChatRedirectFeature.java

```diff
@@ -773,28 +772,6 @@ protected StyledText getNotification(Matcher matcher) {
         }
     }
 
-    private class ScrollTeleportationMobFailRedirector extends SimpleRedirector {
-        private static final Pattern FOREGROUND_PATTERN =
-                Pattern.compile("§cThere are aggressive mobs nearby\\.\\.\\.$");
-
-        @Override
-        protected Pattern getForegroundPattern() {
-            return FOREGROUND_PATTERN;
-        }
-
-        @Override
-        public RedirectAction getAction() {
-            return scrollTeleport.get();
-        }
-
-        @Override
-        protected StyledText getNotification(Matcher matcher) {
-            return StyledText.fromComponent(
-                    Component.translatable("feature.wynntils.chatRedirect.scrollTeleport.notificationMobs")
-                            .withStyle(ChatFormatting.DARK_RED));
-        }
-    }
-
```

Comment:
This message doesn't seem to exist anymore as with the new scroll system the teleport is cancelled if you take damage